package.json

```diff
@@ -114,10 +114,9 @@
       "@semantic-release/commit-analyzer",
       "@semantic-release/release-notes-generator",
       "@semantic-release/changelog",
-      "@semantic-release/git",
-      "@semantic-release/npm"
+      "@semantic-release/git"
     ],
-    "branch": "master"
+    "branch": "release-staging"
```

Comment:
how would you keep `release-staging` up to date with master?

Reply:
I was hoping we can use the "merge latest from mater" feature available in pull requests. Do you think that is viable?

Reply:
sounds good